index.js

```diff
@@ -1 +1,120 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db');
+const server = express();
+server.use(express.json());
+
+// Delivers 'server running' response:
+server.get('/', (req, res) => {
+  res.send('Server running...');
+});
+
+// Fetches users array:
+server.get('/api/users', (req, res) => {
+  db.find()
+    .then(users => {
+      if (users) {
```

Comment:
good thinking for edge cases, but I wouldn't do this for the GET all request, because you could still return the empty array instead which would tell the user more about the data coming back empty than a 404 message